docs/experiment/general/data-model.md

```diff
@@ -12,23 +12,25 @@ At the top level in Amplitude is your **organization**. Within an organization,
+In Amplitude Experiment, a deployment serves a group of flags or experiments for use in an application. Deployments have an associated randomly generated **deployment key** (also known as an API key) which Experiment uses to identify the deployment and authorize requests to the evaluation servers. 
 
 !!!info "Client vs Server Deployments"
-    Deployments are either client or server deployments. Only server deployments can utilize access flag configs for [local evaluation](./evaluation/local-evaluation.md), and should not be shared to made public in any way.
+    Deployments are either client or server deployments. Only server deployments can use access flag configs for [local evaluation](./evaluation/local-evaluation.md), and shouldn't be shared to made public in any way.
 
-Deployments live within Amplitude Analytics projects; a project may have multiple deployments. Deployments are [added to Flags and Experiments](../guides/getting-started/create-a-flag.md#add-a-deployment) which exist within the same project. When a request to fetch variants for a user is received by Experiment's evaluation servers, the deployment key is used to look up all associated flags and experiments for evaluation.
+Deployments live within Amplitude Analytics projects, and a project can have multiple deployments. 
+Add deployments to [Flags and Experiments](../guides/getting-started/create-a-flag.md#add-a-deployment) in the same project. When Experiment's evaluation servers receive a request to fetch variants for a user, Experiment uses the deployment key to look up all associated flags and experiments for evaluation.
 
 ## Flags and experiments
 
-Feature flag and experiments are used to serve a variable experience to a user. They're identified by the **flag key**, associated with `0-n` [deployments](#deployments), and contains `1-k` [variants](#variants). Additionally, the **evaluation mode** (local or remote) determines whether or not the flag or experiment can be [locally evaluated](evaluation/local-evaluation.md) and may limit the targeting capabilities for the flag if set to local (local evaluation mode flags cannot utilize advanced targeting features like behavioral cohorts).
+Experiment uses feature flag and experiments to serve a variable experience to a user. They're identified by the **flag key**, associated with `0-n` [deployments](#deployments), and contains `1-k` [variants](#variants). The **evaluation mode** (local or remote) determines whether the flag or experiment can be [locally evaluated](evaluation/local-evaluation.md) and may limit the targeting capabilities for the flag if set to local (local evaluation mode flags can't use advanced targeting features like behavioral cohorts).
```

Comment:
@bgiori : "They're identified by the **flag key**, associated with `0-n` [deployments](#deployments), and contains `1-k` [variants](#variants)." 

What is "they're" referring to in this sentence: feature flags/experiments, users, or variable experiences?

Reply:
Nice catch, in this case it refers to "Feature flags and experiments".